docs/content/latest/quick-start/build-apps/java/ysql-ebeans.md

```diff
@@ -0,0 +1,369 @@
+
+## Prerequisites
+
+This tutorial assumes that you have:
+
+- YugabyteDB up and running. If you are new to YugabyteDB, follow the steps in [Quick start](../../../../quick-start/) to have YugabyteDB up and running in minutes.
+- Java Development Kit (JDK) 1.8 is installed. JDK installers for Linux and macOS can be downloaded from [OpenJDK](http://jdk.java.net/), [AdoptOpenJDK](https://adoptopenjdk.net/), or [Azul Systems](https://www.azul.com/downloads/zulu-community/).
+- [sbt](https://www.scala-sbt.org/1.x/docs/) is installed.
+
+## Clone the "orm-examples" repository
+
+```sh
+$ git clone https://github.com/yugabyte/orm-examples.git
+```
+
+## Database configuration
+
+- Modify the database configuration section to include YugabyteDB's JDBC driver settings in `conf/application.conf` file as follows:
+
+  ```sh
+  #Default database configuration using PostgreSQL database engine
+  default.username=yugabyte
+  default.password=""
+  default.driver=com.yugabyte.Driver
+  default.url="jdbc:yugabytedb://127.0.0.1:5433/ysql_ebeans?load-balance=true"
+  ```
+
+- Add a dependency in `build.sbt` for YugabyteDB’s JDBC driver.
+
+  ```sh
+  libraryDependencies += "com.yugabyte" % "jdbc-yugabytedb" % "42.3.0"
+  ```
+
+- From your local YugabyteDB installation directory, connect to the [YSQL] shell to create a database.
+
+  ```sh
+  bin/ysqlsh
+  CREATE DATABASE ysql_ebeans;
+  ```
+
+## Build the application
+
+Build the REST API server using:
+
+```sh
+$ sbt compile
+```
+
+{{< note title="Note" >}}
+
+- Certain subversions of JDK 1.8 would require the `nashorn` package. Add the package in the `build.sbt` if there is a compile error regarding a non-existing `jdk.nashorn` package.
+
+```sh
+libraryDependencies += "com.xenoamess" % "nashorn" % "jdk8u265-b01-x3"
+```
+
+- To change the default port(8080) for the REST API Server, set `PlayKeys.playDefaultPort` value in the `src/build.sbt` file.
+{{< /note >}}
+
+## Run the application
+
+```sh
+$ sbt run
+```
+
+## Send requests to the application
+
+Create 2 users.
+
+```sh
+$ curl --data '{ "firstName" : "John", "lastName" : "Smith", "email" : "jsmith@example.com" }' \
+   -v -X POST -H 'Content-Type:application/json' http://localhost:8080/users
+```
+
+```sh
+$ curl --data '{ "firstName" : "Tom", "lastName" : "Stewart", "email" : "tstewart@example.com" }' \
+   -v -X POST -H 'Content-Type:application/json' http://localhost:8080/users
+```
+
+Create 2 products.
+
+```sh
+$ curl \
+  --data '{ "productName": "Notebook", "description": "200 page notebook", "price": 7.50 }' \
+  -v -X POST -H 'Content-Type:application/json' http://localhost:8080/products
+```
+
+```sh
+$ curl \
+  --data '{ "productName": "Pencil", "description": "Mechanical pencil", "price": 2.50 }' \
+  -v -X POST -H 'Content-Type:application/json' http://localhost:8080/products
+```
+
+Create 2 orders.
```

Comment:
Adding orders not working

Note: Unnecessary use of -X or --request, POST is already inferred.
*   Trying ::1...
* TCP_NODELAY set
* Connected to localhost (::1) port 8080 (#0)
> POST /orders HTTP/1.1
> Host: localhost:8080
> User-Agent: curl/7.64.1
> Accept: */*
> Content-Type:application/json
> Content-Length: 65
> 
* upload completely sent off: 65 out of 65 bytes
< HTTP/1.1 404 Not Found
< Referrer-Policy: origin-when-cross-origin, strict-origin-when-cross-origin
< X-Frame-Options: DENY
< X-XSS-Protection: 1; mode=block
< X-Content-Type-Options: nosniff
< X-Permitted-Cross-Domain-Policies: master-only
< Date: Thu, 24 Feb 2022 16:18:55 GMT
< Content-Type: text/plain; charset=UTF-8
< Content-Length: 23
< 
* Connection #0 to host localhost left intact
User id does not exist.* Closing connection 0

Reply:
Confirmed with @priyanshi-yb regarding this issue. So the user Id is auto-generated , so we felt it would be better to verify what the user id is and specify that value in the orders curl command. I'll include these changes.